src/reader/async_tokio.rs

```diff
@@ -81,19 +79,17 @@ impl<R: AsyncBufRead + Unpin> Reader<R> {
     /// ```
     ///
     /// [`read_event_into()`]: Reader::read_event_into
-    pub fn read_event_into_async<'reader, 'b: 'reader>(
+    pub async fn read_event_into_async<'reader, 'b: 'reader>(
```

Comment:
Remove lines 31-38 from the docstring, as it no longer relevant.

Remove the `'reader` lifetime as there no reasons to name it explicitly